src/Nethermind/Nethermind.Runner/configs/mainnet.cfg

```diff
@@ -6,6 +6,10 @@
     "LogFileName": "mainnet.logs.txt",
     "MemoryHint": 2048000000
   },
+  "TxPool": {
+    "Size": 1024,
+    "BlobsSupport": "StorageWithReorgs"
```

Comment:
If we are going to mainnet, I think we should make "StorageWithReorgs" default, and actually set it different only in places where we don't need it.